src/elements/swapMarket/SwapMarket.tsx

```diff
@@ -368,7 +370,14 @@ export const SwapMarket = ({
                   )}
                 </div>
                 <div className="flex justify-between">
-                  <span>Price Impact</span>
+                  <div className="flex items-center">
+                    <span>Price Impact</span>
+                    <Tooltip
+                      button={<IconInfo className="w-12 mx-[3px]" />}
```

Comment:
maybe we can remove the default sizing from the button prop in `Tooltip.tsx` so it defaults to 100% of the parent container. Then we can just wrap the tooltip - might be a bit cleaner than importing the icon component again.

Reply:
Thanks for the review @pingustar and nice word at the beginning.

Not sure about this comment, can we maybe discuss on telegram?
Please DM me on @baart1989 or let me know your name?

Cheers